data/functions/itemized.sql

```diff
@@ -88,6 +92,10 @@ BEGIN
         EXECUTE format('CREATE INDEX idx_%s_sub_id_line_num_dt%s ON %I (line_num, contb_receipt_dt, sub_id)', child_table_root, index_name_suffix, child_table_name);
         EXECUTE format('CREATE INDEX idx_%s_two_year_transaction_period_dt%s ON %I (two_year_transaction_period, contb_receipt_dt, sub_id)', child_table_root, index_name_suffix, child_table_name);
 
+        EXECUTE format('CREATE INDEX idx_%s_contributor_name_text%s ON %I USING GIN (contributor_name_text, contb_receipt_dt, sub_id)', child_table_root, index_name_suffix, child_table_name);
+        EXECUTE format('CREATE INDEX idx_%s_contributor_employer_text%s ON %I USING GIN (contributor_name_text, contb_receipt_dt, sub_id)', child_table_root, index_name_suffix, child_table_name);
```

Comment:
I don't think they are, I had modified these to have `contb_receipt_dt` mixed with them, and the ones below now have `contb_receipt_amt` mixed in them (for schedule A).